scripts/GetClrDbg.ps1

```diff
@@ -104,13 +104,14 @@ if (-not $RuntimeID) {
 }
 Write-Host "Info: Using Runtime ID '$RuntimeID'"
 
-# create the install folder if it does not exist
-if (-not (Test-Path -Path $InstallPath -PathType Container)) {
-    New-Item -ItemType Directory -Force -Path $InstallPath
+# create the temp folder if it does not exist
+$TempPath = Join-Path -Path $env:TEMP -ChildPath "getclrdbg"
+if (-not (Test-Path -Path $TempPath -PathType Container)) {
+    New-Item -ItemType Directory -Force -Path $TempPath
 }
```

Comment:
Got it, I see above it will override the contents regardless. Is there any other file that may be left behind that could be a problem? If not then no need to delete the folder.